app/models/member.rb

```diff
@@ -20,14 +20,13 @@ class Member < ApplicationRecord
   has_many :notes, as: :notable
 
   PRONOUNS = ["he/him", "she/her", "they/them"]
-  enum pronoun: [:"he/him", :"she/her", :"they/them", :custom_pronoun]
+  enum pronoun: [:"he/him", :"she/her", :"they/them"]
```

Comment:
I think we can remove this line entirely since we're removing the `pronoun` column.

Reply:
Make sense 👍🏻